langchain/chains/sql_database/exception_handler.py

```diff
@@ -0,0 +1,40 @@
+"""
+TODO: Depending on how many exceptions end up being handled, this could be
+moved to a method in the SQLDatabaseChain class.
+"""
+
+from sqlalchemy.exc import InvalidRequestError
+from langchain.chains.sql_database.base import SQLDatabaseChain
+from langchain.chains.llm import LLMChain
+
+class ExceptionHandler:
```

Comment:
I think we might be able to move this logic into the `SQLDatabaseChain` class! Let's chat offline a bit